test/source/tests/settings.ts

```diff
@@ -60,13 +54,7 @@ export const defineSettingsTests = (testVariant: TestVariant, testWithBrowser: T
       testWithBrowser(async (t, browser) => {
         const acct = 'flowcrypt.compatibility@gmail.com';
         t.mockApi!.configProvider = new ConfigurationProvider({
-          attester: {
-            pubkeyLookup: {
-              [acct]: {
-                pubkey: somePubkey,
-              },
-            },
-          },
+          attester: singlePubKeyAttesterConfig(acct, somePubkey),
```

Comment:
`somePubkey` isn't related to `flowcrypt.compatibilty@gmail.com` account (or 'ci.tests.gmail` account), but it affects the tests because it may be looked up for signature verification. This complicates tests by adding an unnecessary pubkey into testing.
I see the pubkey returned by the attester in this (and a dozen of other) test only serves as a flag (yes/no). Still it'd be better if a pubkey related to the account is returned.
This isn't hard to implement. we can do a helper method like `setupCommonAcctWithAttester` -- it'll work in a similar way as the current version of `setUpCommonAcct`, but it can also parse the private key, convert it into pubkey with `KeyUtil.asPublicKey` and wire it to the attester config. Even less code in tests.
@sosnovsky ?

Reply:
Sounds good, thanks for proposal 👍